x-pack/legacy/plugins/uptime/public/components/functional/charts/donut_chart_legend.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer } from '@elastic/eui';
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { DonutChartLegendRow } from './donut_chart_legend_row';
+import { UptimeSettingsContext } from '../../../contexts';
+
+const LegendContainer = styled.div`
+  margin-right: 32px;
+  max-width: 150px;
+`;
+
+interface Props {
+  down: number;
+  up: number;
+}
+
+export const DonutChartLegend = ({ down, up }: Props) => {
+  const {
+    colors: { gray, danger },
+  } = useContext(UptimeSettingsContext);
+  return (
+    <LegendContainer>
+      <DonutChartLegendRow
+        color={danger}
+        content={down}
+        message={i18n.translate('xpack.uptime.donutChart.legend.downRowLabel', {
+          defaultMessage: 'Down',
+        })}
+      />
+      <EuiSpacer size="xl" />
```

Comment:
I would update this to a `size="m"`
<img width="289" alt="Screenshot 2019-10-09 at 10 56 19" src="https://user-images.githubusercontent.com/20965076/66466711-72412c80-ea83-11e9-98c8-325f188155b7.png">

Reply:
Fixed in https://github.com/elastic/kibana/pull/47176/commits/8c65f72f90a1af07381c4a7e05044958f4c22214.